Backend/src/controllers/jobController.js

```diff
@@ -84,6 +87,56 @@ job.getAllJobs = async (req, res, next) => {
   } 
 };
 
+//upload document for jobs
+/* { HTTP METHOD - POST, API ENDPOINT - http://localhost:3336/document/uploadDocument }*/
+job.uploadDocument = [
+  upload.single('file_path'), // Multer middleware to handle single file upload
+  async (req, res, next) => {
+    try {
+
+      //Check if the document already exists for the job
+      let checkDoc = await documentModel.findOne({
+        where: {
+          job_id: req.body.job_id,
+        },
+        raw: true
+      });
+
+      if (checkDoc) {
```

Comment:
are we not allowing multiple uploads

Reply:
At this stage, the current implementation supports only a single document upload. The primary goal was to verify the functionality and ensure the feature works as expected.

We plan to extend the functionality to allow multiple document uploads in the future. For now, this approach serves as a foundational step to validate the upload process.

Let me know if there are any immediate concerns or suggestions for this approach!

Reply:
Alright. Approved